src/main/kotlin/no/nav/tilleggsstonader/sak/opplysninger/søknad/mapper/SøknadsskjemaBarnetilsynMapper.kt

```diff
@@ -60,6 +61,14 @@ object SøknadsskjemaBarnetilsynMapper {
                     data =
                         BarnMedBarnepass(
                             type = it.type.verdi,
+                            utgifter =
+                                it.utgifter?.let { utgifter ->
+                                    Utgifter(
+                                        harUtgifterTilPass = utgifter.harUtgifterTilPass.verdi,
+                                        fom = utgifter.fom?.verdi,
+                                        tom = utgifter.tom?.verdi,
+                                    )
+                                },
```

Comment:
Hva med å legge til i søknadBarn.kt:
```
import no.nav.tilleggsstonader.kontrakter.søknad.barnetilsyn.Utgifter as UtgifterKontrakter

fun UtgifterKontrakter?.tilDomene(): Utgifter? =
    this?.let {
        Utgifter(
            harUtgifterTilPass = it.harUtgifterTilPass.verdi,
            fom = it.fom?.verdi,
            tom = it.tom?.verdi,
        )
    }
```

også i denne filen bruke: 
```suggestion
                            utgifter = it.utgifter.tilDomene()
```

Reply:
Fikset.

Reply:
Godt forslag fra @sberbom, men burde vi la denne være i Mapping-filen der all annen mapping fra kontrakter til domenet er? :) Så får vi litt cleanere domene-fil

Reply:
Flyttet mapping logikken fra SøknadBarn.kt til SøknadsskjemaBarnetilsynMapper